### PR TITLE
Add Python3 CLI Commands

### PR DESCRIPTION
**Pull request by sushil490023**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [x] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repository and upgrade the version in the pull request but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
automation

**Comment by wangzelin007** [comment]
Hi @sushil490023,
May I ask you to add some scenario tests for those new commands?

File: src/automation/azext_automation/aaz/latest/automation/python3package/_create.py [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "automation python3package create",
```
Comment (zhoxing-ms) [inline src/automation/azext_automation/aaz/latest/automation/python3package/_create.py:15]: Please separate different words with `-`
```suggestion
    "automation python3-package create",
```

Comment (zhoxing-ms) [reply src/automation/azext_automation/aaz/latest/automation/python3package/_create.py:15]: In addition, if we want to expand multiple different package types in the future, could we design this command as `automation package create` and `python3` as one of the valid values of the `--package-type` parameter? 
This design may have better scalability

**Review by wangzelin007 — COMMENTED** [review]
Hi @sushil490023,
Please also add example for AAZCommand like:
https://github.com/Azure/azure-cli/blob/75e419d2aa01337891914e477680919ff92c193f/src/azure-cli/azure/cli/command_modules/monitor/aaz/latest/monitor/log_analytics/cluster/_show.py#L17-L22

**Comment by sushil490023** [comment]
Created New PR as was facing issue while generating commands for Python3package. Starting from scratch.

PR : https://github.com/Azure/azure-cli-extensions/pull/5415

File: src/automation/azext_automation/aaz/latest/automation/python3package/_create.py (hunk 1 above)
Comment (sushil490023) [reply src/automation/azext_automation/aaz/latest/automation/python3package/_create.py:15]: I have changed name as suggested

**Comment by sushil490023** [comment]
> Hi @sushil490023, Please also add example for AAZCommand like: https://github.com/Azure/azure-cli/blob/75e419d2aa01337891914e477680919ff92c193f/src/azure-cli/azure/cli/command_modules/monitor/aaz/latest/monitor/log_analytics/cluster/_show.py#L17-L22

Added examples for each commands

**Comment by sushil490023** [comment]
> Hi @sushil490023, May I ask you to add some scenario tests for those new commands?

Added scenario test also. Thanks

**Pull request closed without merging**